tests/upgrades/test_performance_tuning.py

```diff
@@ -327,7 +327,7 @@ def test_post_performance_tuning_apply(self, default_sat):
             )
             command_output = default_sat.execute(
                 'satellite-installer --tuning default -s --disable-system-checks',
-                timeout=1000,
+                timeout=1000000,
```

Comment:
why so big timeout ? I would see 3000 is enough

Reply:
The units changed here - but I think you're suggesting changing timeout from `1000` to `300` (old units). In the new ssh2 timeout units, this would be `300000`

Reply:
in that commit, i explain that paramiko timeouts are in seconds while ssh2-python timeouts are in milliseconds.

Reply:
okay my bad I thought about that if it is unit change, i'm pretty good with that